docs/apis/group.yaml

```diff
@@ -374,6 +346,66 @@ paths:
                     code: "GRP-1006"
                     message: "Cannot delete group"
                     description: "Cannot delete group with child groups"
+        "500":
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+              example: "Internal server error"
+
+  /groups/{id}/members:
+    get:
+      tags:
+        - groups
+      summary: List members in group
+      parameters:
```

Comment:
Would it make sense to have a filter parameter to specify which members to return? I.e. either users, groups, or both